sdk_lib/make_chroot.sh

```diff
@@ -47,6 +47,11 @@ DEFINE_string stage3_path "" \
   "Use the stage3 located on this path."
 DEFINE_string cache_dir "" "Directory to store caches within."
 
+DEFINE_string extra_portdir_overlay_path "" \
+  "Extra PORTDIR_OVERLAY direcory path outside of the chroot."
+DEFINE_string extra_portdir_overlay_name "" \
```

Comment:
I did contemplate something like this, just specifying the name of the overlay and then assuming that it is located in ```src/third_party``` as ```src/third_party/overlay-name``` but again, I was trying to not 'over assume' since despite building VMs for months, I haven't needed to tamper with any of this code at all before.